packages/app/src/templates/components/template-form/template-form.component.test.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { waitFor } from "@testing-library/dom"
+import { fireEvent } from "@testing-library/react"
+import { noop } from "Renderer/utils/noop"
+import { renderWithThemeAndIntl } from "Renderer/utils/render-with-theme-and-intl"
+import { TemplateForm } from "App/templates/components/template-form/template-form.component"
+import { TemplateFormProps } from "App/templates/components/template-form/template-form.interface"
+import { TemplateFormTestIds } from "App/templates/components/template-form/template-form-ids.enum"
+
+const onCloseMock = jest.fn()
+const onSaveMock = jest.fn()
+const textMock = "Luke, I'm your father"
+const longTextMock =
+  "Duis tincidunt dui ut condimentum sagittis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris convallis neque eget mi posuere porta ut ut felis. Fusce ultrices cursus interdum. Nulla nibh eros, egestas eu felis vel, posuere pretium elit. Nullam vitae luctus eros, eu lobortis augue. Vestibulum in felis pharetra, lacinia nibh vitae, vehicula velit. Quisque massa est, placerat sed urna vitae, eleifend blandit sapien. Donec a efficitur nunc, et euismod tellus."
+
+const render = (props: TemplateFormProps) => {
+  return renderWithThemeAndIntl(<TemplateForm {...props} />)
+}
+
+const renderWithWaitForm = async (props: TemplateFormProps) => {
+  const outcome = render(props)
+  await waitFor(noop)
```

Comment:
This is needed? I see that tests pass without this waitFor? 🤷‍♂️ 🤔

Reply:
I just inspired by this https://github.com/mudita/mudita-center/blob/6bc7d468bf70809615f10acec568949b1ceb87fb/packages/app/src/contact-support/components/contact-support-modal.component.test.tsx#L37